specification/src/main/asciidoc/platform/PlatformOverview.adoc

```diff
@@ -199,8 +199,10 @@ bean container.
 ==== Container Requirements
 
 This specification requires that containers
-provide a Java Compatible™ runtime environment, as defined by the Java
-Platform, Standard Edition, v8 specification (Java SE). The applet
+support execution in a Java™ runtime environment, as defined by the Java
+Platform, Standard Edition, v11 specification (Java SE 11). Containers may additionally choose
+to support execution in another Java™ runtime environment, as defined by a Java
+Platform, Standard Edition, v8 or later specification. The applet
```

Comment:
I want to call out my addition of "or later" here. Section 6.1.1 already states that later JDKs can be used (https://github.com/eclipse-ee4j/jakartaee-platform/pull/302/files#diff-5be160f30791b49f1b2029c53011cadb84d7f44935afa25337e085e51e118b25R38) so my intent here was to bring this section into alignment.

Reply:
Good addition.  Thanks.